index.html

```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="techlist">
+                    <li class="tech">HTML</li>
+                    <li class="tech">CSS</li>
+                    <li class="tech">JavaScript</li>
+                </ul>
+                <button class="project">See project</button>
+            </div>
+        </section>
+    </div>
+    <div class="about-me">
+        <section class="about">
+            <h1>About Myself</h1>
+            <p>
+                Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <h2>LET'S CONNECT</h2>            
+            <ul class="social">
+                <li>
+                    <a href="https://twitter.com/FirePicaso">
+                        <img src="Assets/Twitter.png" alt="">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/mustakim-masum/">
+                        <img src="Assets/Linkedin.png" alt="">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.monster.com/">
+                        <img src="Assets/Monster.png" alt="">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/firepicaso">
+                        <img src="Assets/gitHub.png" alt="">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.google.com">
+                        <img src="Assets/fingers.png" alt="">
+                    </a>
+                </li>                
+            </ul>
+            <button class="resume">Get my resume</button>
+            <h6>Languages</h6>
+            <ul class="technologies">
+                <li class="techlist">
+                    <div>
+                        <img src="Assets/JS.png" alt="JavaScript">
+                    </div>
+                    <div>
+                        <h3>JavaScript</h3>
+                    </div>
+                </li>
+                <li class="techlist">
+                    <div>
+                        <img src="Assets/HLML.png" alt="HTML">
+                    </div>
+                    <div>
+                        <h3>HTML</h3>
+                    </div>
+                </li>
+                <li class="techlist">
+                    <div>
+                        <img src="Assets/CSS.png" alt="CSS">
+                    </div>
+                    <div>
+                        <h3>CSS</h3>
+                    </div>
+                </li>
+            </ul>
+            <h6>Frameworks</h6>
+            <h6>Skills</h6>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please use `ul` and `li` for categories in "About myself" section. i.e. `Languages`, `Frameworks`, and `Skills` should be list items.


Example
```HTML
<ul>
  <li>
    <h3>Languages</h3>
    . . . 
    <ul>
       <li>JavaScript</li>
       <li> . . .  </li>
       <li> . . .  </li>
    </ul>
  </li>
  <li>
    <h3>Frameworks</h3>
     . . . 
  </li>
  <li> 
   <h3>Skills</h3>
    . . . 
  </li>
</ul>
```

Your design
![image](https://github.com/firepicaso/RWD_Portfolio/assets/78251927/1eaa5afb-8d5e-4d00-8fe1-b7be9566281c)

Figma
![image](https://github.com/firepicaso/RWD_Portfolio/assets/78251927/4e21d498-9342-4be5-9428-bbfb49be42ad)

Kindly adjust your design to ensure that your design is similar to the guideline as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💪

Reply:
I am very close to this template.. also running out of time. @abdulhamiid